tests/s3/test_object_tagging.py

```diff
@@ -1387,3 +1388,380 @@ def test_get_non_existing_object_tagging_2487(self):
                 error.message), error.message
         self.log.info(
             "Verify get object with tagging support to non-existing object")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_tags
+    @pytest.mark.tags("TEST-42774")
+    def test_multipartupload_max_tags_42774(self):
+        """Verify Multipart Upload with max no. of tags"""
+        self.log.info(
+            "Verify Multipart Upload with max no.of tags for an Object")
+        self.log.info("Creating a bucket with name %s",
+                      self.bucket_name)
+        resp = self.s3_test_obj.create_bucket(
+            self.bucket_name)
+        assert resp[0], resp[1]
+        assert resp[1] == self.bucket_name, resp[1]
+        self.log.info(
+            "Bucket is created with name %s",
+            self.bucket_name)
+        self.log.info("Creating multipart upload with tagging")
+        resp = self.tag_obj.create_multipart_upload_with_tagging(
+            self.bucket_name,
+            self.object_name,
+            S3_OBJ_TST["test_9437"]["object_tag"])
+        assert (resp[0]), resp[1]
```

Comment:
For code consistency, better to use assert from assert_utils.

Reply:
Updated with using assert_utils.